support-frontend/assets/components/ticker/contributionTicker.jsx

```diff
@@ -45,74 +64,114 @@ export default class ContributionTicker extends Component<PropTypes, StateTypes>
 
     this.tickerJsonUrl = props.tickerJsonUrl;
     this.onGoalReached = props.onGoalReached;
+    this.tickerType = props.tickerType;
+    this.classModifiers = [this.tickerType];
+    this.increaseTextCounter = this.increaseTextCounter.bind(this);
+    this.goalReached = false;
+
+    this.dataFromServer = {
+      totalContributed: null,
+      goal: null,
+    };
 
     this.state = {
-      totalSoFar: 0,
+      count: 0,
       goal: 0,
     };
 
   }
 
   componentDidMount(): void {
-    getInitialTickerValues(this.tickerJsonUrl).then(({ totalSoFar, goal }) => {
-      const initialTotal = totalSoFar * 0.8;
-      this.count = initialTotal;
-      this.totalSoFar = totalSoFar;
+    getInitialTickerValues(this.tickerJsonUrl).then(({ totalContributed, goal }) => {
+      const initialCount = totalContributed ? totalContributed * 0.8 : 0;
```

Comment:
oh, and another brief comment about why we're starting from 80% would be helpful

Reply:
👍

Reply:
```
      /* ***************************************************************
      Starting the initial count at 80% of its total instead of 0
      affects the increaseTextCounter animation by making it shorter &
      therefore more performant. Animating from 0 to £50,000 (for instance)
      would take significantly longer than we want to run this animation.
      By running it for a short period, we give the experience of the
      number going up as if in real time without taking a big performance
      hit or making the user bored
      ***************************************************************** */
```